packages/components/organisms/f-checkout/src/components/tests/Checkout.test.js

```diff
@@ -18,9 +21,10 @@ describe('Checkout', () => {
     allure.feature('Checkout');
     const checkoutUrl = 'http://localhost/checkout';
     const checkoutAvailableFulfilmentUrl = 'http://localhost/checkout/fulfilment';
+    const loginUrl = 'http://dummy-login.example.com';
 
     it('should be defined', () => {
-        const propsData = { checkoutUrl, checkoutAvailableFulfilmentUrl };
+        const propsData = { checkoutUrl };
```

Comment:
Are you sure you need to remove the existing URL being passed? This doesn't look right.

Reply:
No, good spot.